hw_01/part_1/js/script.js

```diff
@@ -0,0 +1,50 @@
+//NUMBERS
+
+alert(Math.PI.toFixed(2)); //"3.14"
+
+var num = 0.6 + 0.7;
+if (num == 1.3) {
+  alert('Вираз "0,6 + 0,7" дорівнює значенню 1,3');
+} else {
+  alert('Вираз "0,6 + 0,7" не дорівнює значенню 1,3');
+}
+
+var num = 0.6 + 0.7;
+alert(num.toFixed(1));
+
+alert(parseInt("100$"));
+
+//STRINGS
+
+var str = "some test string";
+alert('"' + str[0] + '"' + " and " + '"' + str[str.length - 1] + '"');
+
+var str = "some test string";
+alert(
+  str.slice(0, 1).toUpperCase() +
+    str.slice(1, str.length - 1) +
+    str.slice(str.length - 1).toUpperCase()
+);
+
+var str = "some test string";
+alert(str.indexOf(" ")); // перший пробіл - 4
+alert(str.indexOf(" ", "5")); // другий пробіл, відраховуючи від 5 символу - 9
```

Comment:
Your logic will not work if somebody change a string. Your logic must adapt to the different situation. The string can be like `"Very longlonglogn string"`  or `"Exampleofstring w i t h o u t normalspace"` etc.